### DISCUSSION TITLE
running non-docker jobs

### DISCUSSION THREAD

**Discussion by fakela** [bacalhau-project/bacalhau#1683] [Q&A] [answered]
Is it possible to use a non-docker job when running Bacalhau

**Comment by enricorotundo** [comment]
Not sure if I fully understand the question but if by non-docker you're fine with a WASM job then `bacalhau wasm` is what you're looking for ;)

**Comment by simonwo** [comment, marked as answer]
Yes! You can run programs using WebAssembly instead. See [the onboarding WebAssembly docs](https://docs.bacalhau.org/getting-started/wasm-workload-onboarding/) for information on how to do that.

In a nutshell:

    bacalhau wasm run /path/to/my.wasm
